app/src/docs/Grid/Grid_AlignBottomEnd.tsx

```diff
@@ -0,0 +1,42 @@
+/* @license Copyright 2023 @paritytech/polkadot-cloud authors & contributors
+SPDX-License-Identifier: GPL-3.0-only */
+
+import { SimpleEditor } from "../lib/SimpleEditor";
+import { Grid } from "@packages/cloud-react/lib/core/Grid";
+
+export const Grid_AlignBottomEnd = () => {
+  const code = `<Grid row alignItems="flex-end">
+  <Grid style={{ height: "fit-content" }} column sm={12} md={2}>1/6 and bottom</Grid>
+</Grid>`;
+
+  const separatorStyle = {
+    border: "0.1rem dashed var(--border-secondary-color)",
+    padding: "0.5rem",
+    TextAlign: "center",
+  };
+
+  return (
+    <>
+      <div className="grid-demo">
+        <Grid
+          row
+          style={Object.assign({}, separatorStyle, { height: "10rem" })}
+          alignItems="flex-end"
+          justify="flex-end"
+        >
+          <Grid
+            style={Object.assign({}, separatorStyle, {
+              height: "4rem",
+            })}
+            column
+            sm={12}
+            md={2}
+          >
+            <h3>1/6 and bottom</h3>
```

Comment:
Perhaps change these `h3` tags to `h5` for now, they are rather large and are the same styling as the headers currently.

```suggestion
            <h3>1/6 and bottom</h3>
```